wordle/src/main/kotlin/wordle/domain/Game.kt

```diff
@@ -0,0 +1,50 @@
+package wordle.domain
+
+import java.lang.RuntimeException
+
+class Game(
+    private val wordsReader: WordsReader,
+    private val ioProcessor: IOProcessor,
+    private val answerExtractor: AnswerExtractor = FixedAnswerExtractor(),
+    private var judgements: MutableList<Judgement> = mutableListOf(),
```

Comment:
```suggestion
class Game(
    private wordsReader: WordsReader,
    private val ioProcessor: IOProcessor,
    private answerExtractor: AnswerExtractor = FixedAnswerExtractor(),
    private var judgements: MutableList<Judgement> = mutableListOf(),
```

이 두개 파라미터는 필드로 사용되지 않네요!

Reply:
사용하지 않는 부분 제거하겠습니다!